pkg/localcommit/localcommit.go

```diff
@@ -14,6 +14,11 @@ import (
 // https://activestatef.atlassian.net/browse/DX-2524
 var proj *project.Project
 
+type InvalidCommitID struct {
+	*locale.LocalizedError
```

Comment:
You don't need to localize this, since it's being rationalized in captain. There shouldn't be any scenario in which we surface this error directly to the user.